src/Octoshift/Services/GithubApi.cs

```diff
@@ -800,13 +800,19 @@ public virtual async Task<ReattributeMannequinToUserResult> ReclaimMannequinsSki
             variables = new { orgId, sourceId = mannequinId, targetId = targetUserId }
         };
 
-        var response = await _client.PostAsync(url, payload);
-        var data = JObject.Parse(response);
-
-        return data["errors"].Any(x => ((string)x["message"]).Contains("Field 'reattributeMannequinToUser' doesn't exist on type 'Mutation'"))
-            ? throw new OctoshiftCliException("Feature Flag not enabled, ensure Feature Flag is enabled before re-attempting " +
-            "mannequin reattribution with --skip-invitation option")
-            : data.ToObject<ReattributeMannequinToUserResult>();
+        try
+        {
+            return await _retryPolicy.Retry(async () =>
+            {
+                var data = await _client.PostGraphQLAsync(url, payload);
+                return data.ToObject<ReattributeMannequinToUserResult>();
+            });
+        }
+        catch (Exception ex)
+        {
+            throw new OctoshiftCliException($"Feature Flag not enabled, ensure Feature Flag is enabled before re-attempting " +
```

Comment:
I would add a bit more guidance for customers here. If I get this error the first question I'm going to have is "how do I ensure the FF is enabled???"

Reply:
Customers probably shouldn't know that there's a feature flag - that's an implementation detail. I think we should just say something like:

> Reclaiming mannequins with the `--skip-invitation` flag is not enabled for your GitHub organization. For more details, contact GitHub Support.

We expect to turn this flag on for all customers shortly (probably within the next few weeks!), so this should be a short-lived error case anyway.

Reply:
Updated, thanks for the verbiage :)